content/en/docs/setup/_index.md

```diff
@@ -4,11 +4,97 @@ reviewers:
 - erictune
 - mikedanese
 no_issue: true
-weight: 30
 title: Setup
 main_menu: true
+weight: 30
+content_template: templates/concept
 ---
+{% raw  %}
+
+{{% capture overview %}}
+
+You can run Kubernetes almost anywhere, from your laptop to VMs on a cloud
+provider to a rack of bare metal servers. Deciding where to run Kubernetes
+depends on what resources you have available and how much flexibility you need.
+
+Set up a fully-managed cluster by running a single command or craft your own
+customized cluster on your bare metal servers. Use this page to find the type
+of solution that best fits your needs.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Local-machine Solutions
+
+A local-machine solution is a perfect way to get started with Kubernetes. You
```

Comment:
I would avoid superlatives like "perfect" because they are hard to qualify. Perhaps "easy" would be better.